File: write-local-file/src/main/java/io/github/llmagentbuilder/tool/writelocalfile/WriteLocalFileConfig.java
package io.github.llmagentbuilder.tool.writelocalfile;

public class WriteLocalFileConfig {

  private String basePath;

  public String getBasePath() {
    return basePath;
  }

  public void setBasePath(String basePath) {
    this.basePath = basePath;
  }

  @Override
  public String toString() {
    return "WriteLocalFileConfig{" +
        "basePath='" + basePath + '\'' +
        '}';
  }
}

File: write-local-file/src/main/kotlin/io/github/llmagentbuilder/tool/writelocalfile/WriteLocalFileConfig.kt
package io.github.llmagentbuilder.tool.writelocalfile

data class WriteLocalFileConfig(val basePath: String)

data class WriteLocalFileRequest(
    val filename: String,
    val url: String,
    val content: String,
    val append: Boolean,
)

data class WriteLocalFileResponse(
    val path: String,
)